docs/pycharm/instructions.md

```diff
@@ -8,15 +9,34 @@ PyCharm instructions
 **JSON/YAML Setup**
 
 1. Open __Preferences__
-1. Navigate to __Languages and Frameworks__ > __Schemas and DTDs__ > __JSON Schema Mappings__
-1. Add new Mapping (__Ctrl+N__)
-1. Name the Mapping (e.g; "CFN Template Schema")
-1. Enter a Schema URL (e.g; `https://s3.amazonaws.com/cfn-resource-specifications-us-east-1-prod/schemas/2.15.0/all-spec.json`)
-1. Add a mapping to a folder or file path pattern (e.g; `"*-template.json`)
-1. Select the appropriate Schema Version: __JSON schema version 7__
-1. Click __OK__ to save.
-1. Create new file with the extension specified in the mapping (e.g; `my-app-template.json`) 
+2. Navigate to __Languages and Frameworks__ > __Schemas and DTDs__ > __JSON Schema Mappings__
+3. Add new Mapping (__Ctrl+N__)
+4. Name the Mapping (e.g; "CFN Template Schema")
+5. Enter a Schema URL (e.g; `https://s3.amazonaws.com/cfn-resource-specifications-us-east-1-prod/schemas/2.15.0/all-spec.json`)
+6. Add a mapping to a folder or file path pattern (e.g; `"*-template.json`)
+7. Select the appropriate Schema Version: __JSON schema version 7__
+8. Click __OK__ to save.
+9. Create new file with the extension specified in the mapping (e.g; `my-app-template.json`) 
 
-**Gotchas**
+Gotchas
+------
 
 1. PyCharm does not provide the `description` context on mouse-hover (which other IDEs do)
+
+Troubleshooting 
+------
+**JSON Schema not found or contain error in 'all-spec.json': Can not load code model for JSON Schema file 'all-spec.json'**
+
+Pycharm use the `idea.max.intellisense.filesize` platform property to sets the maximum size of files for which PyCharm provides code assistance and to load a JSON schema.
```

Comment:
Just because I'm a pedant, can we update this slightly to;

"PyCharm uses the `idea.max.intellisense.filesize` platform property to set the maximum size of files for which PyCharm provides code assistance and to load a JSON schema."

Reply:
Ahah, changed :)